readme.md

```diff
@@ -64,13 +64,13 @@ For Maven-based projects, add the following to your POM file in order to use Map
 ```xml
 ...
 <properties>
-    <org.mapstruct.version>1.2.0.Final</org.mapstruct.version>
```

Comment:
> One other thing that I noticed is that you changed the 1.2.0.Final into 1.3.0.Final in the Readme. For people seeing the readme it would be a bit confusing when they can't find 1.3.0.Final since it is not released. Maybe we need to do that change once 1.3.0.Final is out or a 1.3.0.Beta2 is out. Otherwise people will get confused.

@filiphr I initially thought this draft would be merged in a dedicated branch, not in master before release. I'm going to revert the changes of this file totally for 1.2.0.Final and keep the Java 1.6 requirement

Reply:
Your initial assumption makes complete sense. Thanks for the revert